### PR TITLE
MetricsSerializer breaks for missing subindicator

### PR DESCRIPTION
**Pull request by Faysalali534**
issue [203]

## Description
There was an Exception raised on not getting the value of  key **subindicators**

## Related Issue
This bug was related to this [issue](https://github.com/OpenUpSA/wazimap-ng/issues/203)
### Updated
`data["subindicators"][subindicator]` was updated by` subindicators_data = data.get("subindicators")` and then by `absolute_value = subindicators_data.get(subindicator)` , now .get will return none in case there is no value found and by default the value will be "N/A" on having no value 

**_Before_**
image

**_After_**
image

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 [issue linked](https://github.com/OpenUpSA/wazimap-ng/issues/203)
- [ ]  📖 changelog filled out

### Commits

- [ ]  commits are clean
- [ ]  commit messages are clean

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: wazimap_ng/profile/serializers/metrics_serializer.py [hunk 1]
```diff
@@ -28,11 +28,20 @@ def sibling(profile_key_metric, geography):
...
+        indicator__profilekeymetrics=profile_key_metric, geography=geography
+    )
     if indicator_data.count() > 0:
         subindicator = get_subindicator(profile_key_metric)
-        data = indicator_data.first().data # TODO what to do with multiple results
-        return data["subindicators"][subindicator]
+        data = indicator_data.first().data  # TODO what to do with multiple results
+        subindicators_data = data.get("subindicators")
+        if subindicators_data:
+            absolute_value = subindicators_data.get(subindicator)
```
Comment (milafrerichs) [inline wazimap_ng/profile/serializers/metrics_serializer.py:39]: You can and should use a default here for `.get` see the docs: https://docs.python.org/3.9/library/stdtypes.html#dict.get

**Review by milafrerichs — CHANGES_REQUESTED** [review]
will you add a test for this?

File: wazimap_ng/profile/serializers/metrics_serializer.py (hunk 1 above)
Comment (Faysalali534) [reply wazimap_ng/profile/serializers/metrics_serializer.py:39]: absolute

correct me if i am wrong, you want this to have default as **'N/A'** and if so then what will be the expected then this

**_if_** below should have extra **_and_** operator for checking instance as dict too? because 'N/A' is string
```
  if subindicators_data:
            absolute_value = subindicators_data.get(subindicator)
```

**Comment by adieyal** [comment]
This issue was addressed by https://github.com/OpenUpSA/wazimap-ng/pull/298
